tubular/imputers.py

```diff
@@ -66,6 +66,32 @@ def _generate_imputation_expressions(self, expr: nw.Expr, col: str) -> nw.Expr:
             else expr
         )
 
+    @staticmethod
+    def _get_all_null_columns(
```

Comment:
Why both with this as a static method? It makes no reference to the object; just have it as a standalone helper function?